src/server/exemption/delete/index.njk

```diff
@@ -0,0 +1,46 @@
+{% extends 'layouts/page.njk' %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "cancel-button/macro.njk" import appCancelButton %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
```

Comment:
Can this be wrapped in a `<nav />` element so it's in a landmark and doesn't fail a11y checks?